### ISSUE TITLE
Not able to pass an empty <string[]> in the ReplicationConfigurationTemplateIDs parameter 

### ISSUE THREAD

**Issue by ShubhamShah11** [aws/aws-tools-for-powershell#256]
### Describe the bug

Issue in Service : DRS
cmdlet : Get-EDRSReplicationConfigurationTemplate
parameter : ReplicationConfigurationTemplateIDs

I am trying to pass an empty list in the Replication configuration Template IDs parameter and it should return all the available Templates but it returns an  "Invalid Request Body" output.

https://docs.aws.amazon.com/powershell/latest/reference/items/Get-EDRSReplicationConfigurationTemplate.html 

### Expected Behavior

It should have returned information about all the available Replication Template Id's

### Current Behavior

It is returning "Invalid Request Body"

Input
```Get-EDRSReplicationConfigurationTemplate -ReplicationConfigurationTemplateIDs @()```

Response
```Get-EDRSReplicationConfigurationTemplate: Invalid request body```

### Reproduction Steps

Run 

Get-EDRSReplicationConfigurationTemplate -ReplicationConfigurationTemplateIDs @() 

### AWS Tools for PowerShell version used

 AWS.Tools.Drs 4.1.68 

### PowerShell version used

Name                           Value
----                           -----
PSVersion                      7.2.2
PSEdition                      Core
GitCommitId                    7.2.2
OS                             Darwin 21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.8…
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

### Operating System and version

MacOs Monterey Version 12.2.1

**Comment by ashishdhingra** [comment]
AWS Tools for PowerShell CmdLets use AWS SDK for .NET behind the scenes. The issue is reproducible using the below code:
```C#
using Amazon.Drs;
using Amazon.Drs.Model;
using System.Collections.Generic;

namespace EDRSReplicationConfigurationTemplate_PWSH_Issue256
{
    class Program
    {
        static void Main(string[] args)
        {
            AmazonDrsClient amazonDrsClient = new AmazonDrsClient();
            var response = amazonDrsClient.DescribeReplicationConfigurationTemplatesAsync(new DescribeReplicationConfigurationTemplatesRequest() { 
                ReplicationConfigurationTemplateIDs=new List<string>()
            }).Result;
        }
    }
}
```

As per documentation for [DRS -> DescribeReplicationConfigurationTemplates](https://docs.aws.amazon.com/drs/latest/APIReference/API_DescribeReplicationConfigurationTemplates.html), the parameter [replicationConfigurationTemplateIDs](https://docs.aws.amazon.com/drs/latest/APIReference/API_DescribeReplicationConfigurationTemplates.html#API_DescribeReplicationConfigurationTemplates_RequestSyntax) is required. But user could pass an empty list which would return all Replication Configuration Templates.

The corresponding AWS CLI command `aws drs describe-replication-configuration-templates --replication-configuration-template-ids` works with the following output:
```JSON
{
    "items": [
        {
            "arn": "arn:aws:drs:us-east-2:139480602983:replication-configuration-template/rct-46d7dc2521f2e634c",
            "associateDefaultSecurityGroup": true,
            "bandwidthThrottling": 0,
            "createPublicIP": true,
            "dataPlaneRouting": "PUBLIC_IP",
            "defaultLargeStagingDiskType": "GP3",
            "ebsEncryption": "DEFAULT",
            "pitPolicy": [
                {
                    "enabled": true,
                    "interval": 10,
                    "retentionDuration": 60,
                    "ruleID": 1,
                    "units": "MINUTE"
                },
                {
                    "enabled": true,
                    "interval": 1,
                    "retentionDuration": 24,
                    "ruleID": 2,
                    "units": "HOUR"
                },
                {
                    "enabled": true,
                    "interval": 1,
                    "retentionDuration": 7,
                    "ruleID": 3,
                    "units": "DAY"
                }
            ],
            "replicationConfigurationTemplateID": "rct-46d7dc2521f2e634c",
            "replicationServerInstanceType": "t3.small",
            "replicationServersSecurityGroupsIDs": [],
            "stagingAreaSubnetId": "subnet-0552f61c30c94db58",
            "stagingAreaTags": {},
            "tags": {},
            "useDedicatedReplicationServer": false
        }
    ]
}
```
If we include template ID `rct-46d7dc2521f2e634c` in the `ReplicationConfigurationTemplateIDs` list in C# code, the logic works, including the PowerShell CmdLet `Get-EDRSReplicationConfigurationTemplate -ReplicationConfigurationTemplateIDs @("rct-46d7dc2521f2e634c")`.

**Root Cause:**
In [DescribeReplicationConfigurationTemplatesRequestMarshaller](https://github.com/aws/aws-sdk-net/blob/master/sdk/src/Services/Drs/Generated/Model/Internal/MarshallTransformations/DescribeReplicationConfigurationTemplatesRequestMarshaller.cs), parameter `replicationConfigurationTemplateIDs` is not included in marshalled request [here](https://github.com/aws/aws-sdk-net/blob/59b09bd880c8e475823bef1c1c89ce459a000cab/sdk/src/Services/Drs/Generated/Model/Internal/MarshallTransformations/DescribeReplicationConfigurationTemplatesRequestMarshaller.cs#L80) since it is an empty list. However, the service API expects this required parameter (even empty one). Hence the error. Using Immediate window expression `snippet="{\"replicationConfigurationTemplateIDs\":[]}"` while Visual Studio debugging to set `snippet` at [line](https://github.com/aws/aws-sdk-net/blob/59b09bd880c8e475823bef1c1c89ce459a000cab/sdk/src/Services/Drs/Generated/Model/Internal/MarshallTransformations/DescribeReplicationConfigurationTemplatesRequestMarshaller.cs#L92) works fine.

Since the code is auto-generated, it is unsure if there is a way/flag in service model that indicates that empty lists should be marshalled.

**Comment by ashishdhingra** [comment]
Fix/workaround has been released in version `4.1.178.0` of AWS Tools for PowerShell.
